jl4-lsp/src/LSP/L4/Viz/Ladder.hs

```diff
@@ -216,27 +248,27 @@ translateDecide (MkDecide _ (MkTypeSig _ givenSig _) (MkAppForm _ funResolved _
   do
     unlessM (hasBooleanType (getAnno body)) $
       throwError InvalidDecideMustHaveBoolRetType
+
+    _              <- collectDefsForInlining
     shouldSimplify <- getShouldSimplify
     vid            <- getFresh
     vizBody        <- translateExpr shouldSimplify body
     pure $ V.MkFunDecl
       vid
       -- didn't want a backtick'd name in the header
-      (mkSimpleVizName funResolved)
+      (mkPrettyVizName funResolved)
```

Comment:
Unrelated refactor: Rename `mkSimpleVizName` to the more accurate `makePrettyVizName`